doc/troubleshooting/model-compatability.md

```diff
@@ -1,5 +1,21 @@
-# Model compatability
+# Model compatibility
 
-When the version of DeePMD-kit used to training model is different from the that of DeePMD-kit running MDs, one has the problem of model compatability.
+When the version of DeePMD-kit used to training model is different from the that of DeePMD-kit running MDs, one has the problem of model compatibility.
 
 DeePMD-kit guarantees that the codes with the same major and minor revisions are compatible. That is to say v0.12.5 is compatible to v0.12.0, but is not compatible to v0.11.0 nor v1.0.0. 
+
+One can execuate `dp convert-from` to convert an old model to a new one.
+
+|   | Model v0.12 | Model v1.0 | Model v1.1 | Model v1.2 | Model v1.3 | Model v2.0 |
+|:-:|:-----------:|:----------:|:----------:|:----------:|:----------:|:----------:|
+|DeePMD-kit v0.12 | 😄 | 😢 | 😢 | 😢 | 😢 | 😢 |
+|DeePMD-kit v1.0  | 😢 | 😄 | 😢 | 😢 | 😢 | 😢 |
+|DeePMD-kit v1.1  | 😢 | 😢 | 😄 | 😢 | 😢 | 😢 |
+|DeePMD-kit v1.2  | 😢 | 😢 | 😢 | 😄 | 😢 | 😢 |
+|DeePMD-kit v1.3  | 😢 | 😢 | 😢 | 😊 | 😄 | 😢 |
```

Comment:
Do we have tools to convert v1.3 model to v1.2 compatibility?

Reply:
No.

Reply:
So why there is a smile? Is it a typo?

Reply:
The smile is where model version is v1.2 and deepmd-kit version is 1.3, not converse.

Reply:
Can we use a v1.2 model in deepmd-kit v1.3?

Reply:
So you mean we can convert a model v1.2 to v1.3 by using deepmd-kit v1.3.   I do not see this subcommand in v1.3

Reply:
We can convert a model v1.2 to v1.3 by using deepmd-kit v1.2. So this should be a 😢?

Reply:
Yes, the manner of conversion is somewhat different... In deepmd-kit v1.2 and 1.3, it is `convert-to` (pending PR #631 and #633), but in v2.0 it is `convert-from`.

I would proposed to only keep the doc for v2.0, and write separate docs for v1.2 and 1.3, what do you think?

Reply:
Good idea.